core/win32/ntdll_shared.c

```diff
@@ -54,7 +54,9 @@
 #    pragma warning(disable : 4204) // nonstd extension: non-constant aggregate
                                     // initializer
 #    define INVALID_FILE INVALID_HANDLE_VALUE
-#    define snprintf _snprintf
+#    if defined(_MSC_VER) && _MSC_VER < 1900
+#        define snprintf _snprintf
```

Comment:
There are many other uses of `_snprintf` in the code base.  Is there a plan to address those?  The commit message does not say why it only picks this use in core/.

Reply:
Probably just because it was the only `snprintf`-related warning to successfully compile using default configuration.
Agree, I should have investigated it better...